synapseAnnotations/data/common/minimal_Sage_standard.json

```diff
@@ -1045,6 +1055,41 @@
         "value": "CD8+ T-Cells",
         "description": "",
         "source": ""
+      },
+      {
+        "value": "CRISPR arachnoid",
+        "description": "",
+        "source": ""
+      },
+      {
+        "value": "primary arachnoid",
```

Comment:
primary is still cultured, but (in most cases) low passage, heterogenous cell mixtures. eg. disaggregated tumor cells being grown in culture, rather than a more homogenous cell "line"